feature/experimental/p4rt/otg_tests/lldp_packetin_test/lldp_packetin_test.go

```diff
@@ -508,3 +515,12 @@ func (lldp *LLDPPacketIO) GetEgressPort() []string {
 func (lldp *LLDPPacketIO) GetIngressPort() string {
 	return lldp.IngressPort
 }
+
+func ateInterface(t *testing.T, topo gosnappi.Config, portID string) gosnappi.Port {
```

Comment:
Remove the t *testing.T parameter? It's unused.

Reply:
Removed the unused parameter